docs/false_positives.md

```diff
@@ -0,0 +1,243 @@
+
+How to suppress a specific dead store warning and more useful tips can be found
+[here](https://clang-analyzer.llvm.org/faq.html).
+
+#### <a name="alternative-implementations"></a> Alternative implementations
+
+If there are huge source of false positives due to the analyzer can not model a
+function properly you could either disable the analyzer to analyze that
+function or provide an alternative implementation that is easier to model
+(i.e.: implementation without bitwise operation tricks). This can be achieved
+using the `__clang_analyzer__` macro.
+
+### <a name="syntax-based-checks"></a> Syntax based checks
+
+Clang tidy has lots of useful syntax based checks. Some of these checks
+find bugprone code snippets. When these snippets are intentional, usually
+there is a natural way to make the intent more explicit. Unfortunately,
+it is hard to give a general guideline, because the details are different for
+each check. The documentation of the check might contain hints how to
+express intention more clearly. Let us look at an example:
+
+  
+```cpp
+double f(int i) {
+  return 32 / (2 + i); // Warning, integer division, loss of precision.
+}
+```
+
+It can be rewritten to as the following to suppress the warning:
+
+```cpp
+double f(int i) {
+  return (int)(32 / (2 + i)); // No warning, the intention is explicit.
+}
+```
+
+The second version makes it clear even though the return value is a floating
+point value the loss of precision during integer division is intentional.
+Adding a comment why this is intentional would make this even clearer.
+Such edits makes the code easier to understand for fellow developers. 
+
+### <a name="suppress-results"></a> Suppress results
+
+When none of the above worked, we can suppress the finding. There are multiple
+ways to do that and it is important to chose the right one.
+
+#### <a name="third-party-code"></a> 3rd party code
+
+We usually have no control over 3rd party code and not interested in the
+findings in such code. CodeChecker supports skip files to skip certain parts
+of the code. This might also make the analysis faster. For details, see the
+[user guide](user_guide.md#skip).
+
+#### <a name="authored-code"></a> Authored code
+
+If you have control over the code we advise to use in source code suppression.
+This has the advantage of the code and the suppression evolving together and
+more reliable than other ways of suppression methods. Only suppress the results
+from one specific check that found the issue. Always comment why a
+finding is considered false positive! With a future version of the analyzer
+these suppression comments might be no longer required. Comments might help in
+the reevaluation. For details, see the [user guide](user_guide.md#suppression-code).
+
+#### <a name="unauthored-code"></a> Unauthored code
+
+If you do not have control over the code for some reason you can suppress 
+issues using the web user interface or the command line client. 
```

Comment:
You can no longer suppress from the command-line client, the importing of old-style suppression files is a legacy migration tool only. Suppression, as a concept, has evolved into the *review status* feature.